package.json

```diff
@@ -71,6 +72,7 @@
   },
   "prettier": "@ionic/prettier-config",
   "dependencies": {
-    "glob": "^10.3.10"
+    "glob": "^10.3.10",
+    "lodash.merge": "^4.6.2"
```

Comment:
I would recommend to use [`deepmerge`](https://www.npmjs.com/package/deepmerge) instead. The lodash package hasn't seen much maintenance recently and maintainer seems to have [given up](https://github.com/lodash/lodash/pulls?q=is%3Apr+is%3Aclosed+label%3A%22issue+bankruptcy%22) on it. I know that there are discussions around who will take this on but I don't know the state of that. Another advantage is that `deepmerge` comes with maintained types built in.

Reply:
Thanks for the suggestion! Went ahead and made that switch, package is smaller too which is nice